### DISCUSSION TITLE
coinbase wallet deprecated?

### DISCUSSION THREAD

**Discussion by j-yw** [wevm/wagmi#350] [Q&A]
Hi amazing people,

I noticed that the coinbase wallet option now shows deprecated, what is the reason for this and what's the solution?

coinbase wallet still supported by wagmi? if so how do I update it to the latest version.

<img width="1720" alt="Screen Shot 2022-04-20 at 21 20 15" src="https://user-images.githubusercontent.com/32629001/164239620-29a42022-ee92-4b29-96e0-6fc05d280cf6.png">

**Comment by tmm** [comment]
It is! Forgot to update the docs: [`WalletLinkConnector`](https://wagmi.sh/docs/connectors/walletLink) is deprecated in favor of [`CoinbaseWalletConnector`](https://wagmi.sh/docs/connectors/coinbaseWallet).

You can upgrade to `wagmi@0.2.28` to use `CoinbaseWalletConnector`.
